accname/name/comp_hidden_not_referenced.html

```diff
@@ -73,6 +73,36 @@
   </span>
 </h2>
 
+<h2>Input with hidden label</h2>
+<label id="hidden-label" for="input-01" hidden>Hidden label</label>
+<input
+  id="input-01"
+  class="ex-label"
+  data-testname="Input that is aria-labelledby hidden label"
+  data-expectedlabel="Hidden label"
+  aria-labelledby="hidden-label"
+>
+
+<h2>Table with hidden caption</h2>
+<table
+  class="ex-label"
+  data-testname="Table that is aria-labelledby hidden caption"
+  data-expectedlabel="Hidden caption"
+  aria-labelledby="hidden-caption"
+>
+  <caption id="hidden-caption" hidden>Hidden caption</caption>
+</table>
+
+<h2>Fieldset with hidden legend</h2>
+<fieldset
+  class="ex-label"
+  data-testname="Fieldset that is aria-labelledby hidden legend"
+  data-expectedlabel="Hidden legend"
+  aria-labelledby="hidden-legend"
+>
+  <legend id="hidden-legend" hidden>Hidden legend</legend>
+</fieldset>
+
```

Comment:
New tests to a file tracked in the current Interop project would only be accepted if they didn't introduce new failures mid-year. In this case, the 3 new tests are all passing in Chromium, Gecko, and WebKit.... (Edge unknown but likely passing?) so I think it's okay to include this in the PR... With the caveat that we could roll it back in the unlikely chance it causes a failure for Edge.

In the future, it may be easier to include Interop-tracked changes as a separate PR from ones that do not affect the current Interop project.

Reply:
Ah, understood, thanks. I’ll make that distinction for future PRs and check in with you if I have doubts.

Reply:
In cases where you need to add a new valid-but-failing tests, add a new file that uses the same file name +".tentative" and have it only include the new failing tests... e.g. `comp_hidden_not_referenced.tentative.html`

Reply:
Got it. 👍🏻  I did just manually confirm that latest Edge _is_ passing all 3 new tests, so... no harm no foul? But I’ll stay ready to roll it back regardless.